src/Microsoft.Health.Dicom.Core/Features/Store/StoreResponseBuilder.cs

```diff
@@ -101,12 +101,11 @@ public StoreResponse BuildResponse(string studyInstanceUid)
         else
         {
             // add comment Sq / list of warnings here
-
-            var warnings = storeValidationResult.Errors.Select(
-                    error => new DicomDataset(
+            var warnings = storeValidationResult.Warnings.Select(
```

Comment:
@lagomesma, The comment says warnings, do we need warning here or the validation errors. Please confirm